acceptance-tests/upgrade/update_and_upgrade_redis_test.go

```diff
@@ -14,16 +14,11 @@ import (
 var _ = Describe("Redis", Label("redis"), func() {
 	When("upgrading broker version", func() {
 		It("should continue to work", func() {
-			if brokerpaks.DetectBrokerpakV140(releasedBuildDir) {
-				Skip("Brokerpak 1.4.0 and earlier no longer can create AWS Redis service instances")
```

Comment:
I am not sure how we are replacing this. Are we skipping somehow differently? Or just getting rid of it as those versions are not longer tested?